ecs-cli/modules/cli/compose/entity/service/service_helper.go

```diff
@@ -15,19 +15,82 @@ package service
 
 import (
 	"fmt"
+	"sort"
+	"time"
 
 	log "github.com/Sirupsen/logrus"
+	ecscli "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/waiters"
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+const (
+	// DefaultUpdateServiceTimeout is the time that the CLI will wait to check if the
+	// count of running tasks is changing. If it has not changed then an error is thrown
+	// after UpdateServiceTimeout minutes
```

Comment:
nit: "DefaultUpdateServiceTimeout" instead of "UpdateServiceTimeout.

Also, I'd say "If _the count_ has not changed..." in line 30.